.github/workflows/swift.yml

```
@@ -39,4 +38,42 @@ jobs:
       - draft-release
     steps:
       - uses: actions/checkout@v3
-      # TODO: Build Apple client from the CLI
+      - uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: ./rust
```

Probably good to apply the fix from #1734 
```suggestion
          workspaces: ./rust
          prefix-key: rust-${{ matrix.runs-on }}
          save-if: ${{ github.ref == 'refs/heads/cloud' }}
```

---

Okay, done.